.github/actions/setup-spack-stack/action.yaml

```diff
@@ -225,11 +284,26 @@ runs:
 
   - name: build-env
     shell: bash
+    env:
+      OMP_NUM_THREADS: 16
```

Comment:
Does this help? The runner only has 2 vcpus, and what uses OpenMP? Does Spack use it internally?

Reply:
OpenMP threads are used for the package-level build parallelism (`config:build_jobs`). It does help, though not a lot. At least it decreases the build time from 6hrs+ to 5hrs 25min with ubuntu/intel for jedi-ewok - good enough for the moment.